pydm/widgets/drawing.py

```diff
@@ -599,6 +609,50 @@ def arrowStartPoint(self, new_selection):
             self._arrow_start_point_selection = new_selection
             self.update()
 
+    @Property(bool)
+    def arrowMidPoint(self):
+        """
+        If True, an arrow will be drawn at the midpoiint of the line.
+        Returns
+        -------
+        bool
+        """
+        return self._arrow_mid_point_selection
+
+    @arrowMidPoint.setter
+    def arrowMidPoint(self, new_selection):
+        """
+        If True, an arrow will be drawn at the midpoiint of the line.
+        Parameters
+        -------
+        new_selection : bool
+        """
+        if self._arrow_mid_point_selection != new_selection:
+            self._arrow_mid_point_selection = new_selection
+            self.update()
+
+    @Property(bool)
+    def arrowMidPointDirection(self):
+        """
+        If True, an arrow will be drawn at the midpoiint of the line.
+        Returns
+        -------
+        bool
+        """
```

Comment:
I might go with `flipMidPointArrow` and `self._mid_point_arrow_flipped`? Or some similar word where it's a bit clearer what will happen when passing True/False to it. But should fix the docstring here and in the setter to describe what it does either way.